src/main/java/com/collections/HashMapExample.java

```diff
@@ -0,0 +1,54 @@
+package com.collections;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class HashMapExample {
+    public static void main(String[] args) {
```

Comment:
Write notes on internal working of hashmap put and get and contains and remove. Very important